app/src/standard/java/com/x8bit/bitwarden/data/platform/manager/CrashLogsManagerImpl.kt

```diff
@@ -32,4 +39,15 @@ class CrashLogsManagerImpl(
         legacyAppCenterMigrator.migrateIfNecessary()
         isEnabled = settingsRepository.isCrashLoggingEnabled
     }
+
+    private inner class NonfatalErrorTree : Timber.Tree() {
+        override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
+            t?.let { trackNonFatalException(BitwardenNonFatalException(message, it)) }
```

Comment:
So I am only logging anything with an actual exception, this will exclude the network logs at the moment. I think that is probably a good thing for now.